tensorflow/lite/micro/micro_allocator_test.cc

```diff
@@ -749,14 +749,14 @@ TF_LITE_MICRO_TEST(TestOperatorInputsNotInSubgraphInputs) {
       48,   // t3
       -1};  // t4
 
-  int t0 = 0;
-  int t1 = 1;
-  int t2 = 2;
-  int t3 = 3;
-  int t4 = 4;
-
-  int num_conns = 2;
-  tflite::testing::NodeConnection node_list[2] = {
+  constexpr int t0 = 0;
+  constexpr int t1 = 1;
+  constexpr int t2 = 2;
+  constexpr int t3 = 3;
+  constexpr int t4 = 4;
+
+  constexpr int num_conns = 2;
```

Comment:
Same comment here - even though this is existing code, would be great if the variable name was spelled out to `number_connections` or similar.

Reply:
Done